### DISCUSSION TITLE
How to drop tagName if tagValue is empty string ?

### DISCUSSION THREAD

**Discussion by sp88011** [NaturalIntelligence/fast-xml-parser#643] [Q&A]
How would I drop a tagName if the tagValue is `""` (=empty string)?

I can't figure out if/how`tagValueProcessor` would help me achieve that because, as per the docs:

```
- If tagValueProcessor returns undefined or null then original value would be set without parsing. <-- problematic
- If it returns different value or value with different data type then new value would be set without parsing.
- Otherwise original value would be set after parsing (if parseTagValue: true)
- if tag value is empty then tagValueProcessor will not be called. <-- problematic
```

Example

```
const xmlDataStr = `
    <root>
        <a></a>
        <b>hello</b>
    </root>`;  
```

Expected output:
```
root: {
b: "hello"
}
```

**Comment by amitguptagwl** [comment]
If you want to try experimental changes for v5, you can use onAttribute override method. You can find the code [here](https://github.com/NaturalIntelligence/fast-xml-parser/tree/master/src/v5). If I write any tutorial, I'll let you know.

**Reply by sp88011** [reply]
Thanks for the hint. I took a look but wasn't able to figure out how to use this to my advantage since there's no attribute on my tag. Anyway, I look forward to v5 and your tutorial :)
